src/nlp.jl

```diff
@@ -218,6 +218,44 @@ function dual(c::NonlinearConstraintRef)
     return nldata.nlconstr_duals[c.index.value]
 end
 
+"""
+    nlp_dual_start_value(model::Model)
+
+Return the current value of the MOI attribute [`MOI.NLPBlockDualStart`](@ref).
+"""
+function nlp_dual_start_value(model::Model)
+    return MOI.get(model, MOI.NLPBlockDualStart())
+end
+
+"""
+    set_nlp_dual_start_value(
+        model::Model,
+        start::Union{Nothing,Vector{Float64}},
+    )
+
+Set the value of the MOI attribute [`MOI.NLPBlockDualStart`](@ref). Pass
+`nothing` to unset a previous start.
```

Comment:
This could use an explanation of what the vector means and how to map it to nonlinear constraints that have been added to the model.